src/main/java/seedu/address/logic/commands/DietCommand.java

```diff
@@ -59,7 +58,7 @@ public CommandResult execute(Model model) throws CommandException {
                 petToEdit.getAttendanceHashMap());
 
         model.setPet(petToEdit, editedPet);
-        model.updateFilteredPetList(PREDICATE_SHOW_ALL_PETS);
+        model.updateFilteredPetList();
```

Comment:
Not too sure about this one as well. Maybe you can add a line `expectedModel.updateFilteredPetList()` after the ` expectedModel.setPet(firstPerson, editedPerson);` line as shown in the image below. 

![image](https://user-images.githubusercontent.com/69782911/161732419-1cdd3317-5a6d-4fa1-9249-6a5ec74dbd61.png)